cartoframes/viz/kuviz.py

```diff
@@ -0,0 +1,47 @@
+from carto.kuvizs import KuvizManager
+from carto.exceptions import CartoException
+
+
+class Kuviz(object):
+    PRIVACY_PUBLIC = 'public'
+    PRIVACY_PASSWORD = 'password'
+
+    def __init__(self, context, id, url, name, privacy=PRIVACY_PUBLIC):
+        self.cc = context
+        self.id = id
+        self.url = url
+        self.name = name
```

Comment:
Do we support any name (spaces, accents, etc.)?

Reply:
I think so. In the db is a text field, do you see any problem?